src/Pages/Index/data.json

```diff
@@ -4,15 +4,16 @@
       {
         "messages": [
           "!help",
-          "!set thanks::you're welcome!",
+          "!set thanks::[you're welcome!, no problem!]",
+          "thanks",
           "thanks",
           "!remove thanks"
         ]
       },
       {
         "messages": [
           "!set hi::hey",
-          "!set hi::hello :wave:",
```

Comment:
The purpose of the duplicate `!set` calls is to show the error that occurs when attempting to override; the repetition/error is intentional.